Application/src/view/visuals/component/Message.java

```diff
@@ -231,11 +237,27 @@ public void setStatic(boolean staticIndicator)  {
     public void renderDefault(GraphicsContext gc) {
 
 
+        if(trails.size() != 0) {
+            Trail first;
```

Comment:
This variable name doesnt have anything to do with the accual variable

Reply:
@timjon i am gonna call it "t" just like @IsabelleTornqvist suggested

Reply:
currentTrail would be better for example since it might be any index in the list